.env.example

```
@@ -87,5 +87,22 @@ NIFI_TIMEOUT=30000
 # Unstructured-Pipeline Configuration
 # -----------------------
 NODE_ENV=dev
-NIFI_URL=http://$${SERVER_C_HOST}:8088/nifi-api
-SOLR_URL=http://$${SERVER_C_HOST}:8983/solr/biar_docs
\ No newline at end of file
+FUNCTION_NAME=biar-processor
+
+# CouchDB Configuration
+COUCHDB_URL=http://simon:1234@10.10.80.16:5984/cms-evidence
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The COUCHDB_URL key is duplicated

(DuplicatedKey)